lib/resources/dstu2/soarian_procedure.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: Procedure
+field_name_base_url: http://hl7.org/fhir/DSTU2/procedure-definitions.html#Procedure
+fields:
```

Comment:
Its an identifier assigned by Soarian, FHIR cardinality allows the “type” as optional.